.gitlab-ci.yml

```diff
@@ -15,7 +15,13 @@ include:
     file: '.gitlab-ci-check-apidocs.yml'
 
 test:static:
+  before_script:
+    - |
```

Comment:
I'd ping Lluis - maybe as a next step we should kick it up to the mendertesting templates.

Reply:
Thanks for reminding me.
@lluiscampos I forgot to bring it up on today's QA review, but I've been using `golangci-lint` for some time locally and I think it has some pretty useful checks (it also wraps the usual static checks: `go vet` and `gocyclo`, in addition to a lot of good stuff).

Reply:
Thanks for involving me.

This looks interesting, indeed, and we should move it into the template. Can we just do it right now? Or do you guys expect some repos to fail?

With regards to this PR, as we are rewriting the whole `test:static` job we should remove the template include.

Reply:
Most repos will definately fail as it is now, my editor usually report a dozen warnings using the 10 default linters. Though many of these warnings are formatting errors, some of them are actually pretty useful, such as unused variables, uncaught errors and http response bodies not closed.
If you think it is as cool as I do, let's create a task for it :+1:

Reply:
@alfrunes Yes, that should be the proper process for doing this. Thanks!